lib/transport_web/controllers/user_controller.ex

```diff
@@ -40,7 +40,7 @@ defmodule TransportWeb.UserController do
       {:ok, response} ->
         conn
         |> put_flash(:info, dgettext("user", "Organization added"))
-        |> redirect(to: user_path(conn, :organization_datasets, response["slug"]))
+        |> redirect(to: user_path(conn, :organization_datasets, response["id"]))
```

Comment:
Same, this is our front's representation, it not the same context than our internal relationship with data.gouv.fr.